benchmark/server/scenarios/system.js

```diff
@@ -12,74 +11,42 @@ const suite = new Benchmark.Suite('system', {
 });
 Benchmark.options.minSamples = 100;
 
-const materialSystem = compose(palette, spacing, typography);
-const styledSystem = compose2(color, space, fontFamily, fontSize);
-
 const materialSystemTheme = createMuiTheme();
 
-const styledSystemTheme = createMuiTheme();
-styledSystemTheme.breakpoints = ['40em', '52em', '64em'];
-styledSystemTheme.colors = styledSystemTheme.palette;
-styledSystemTheme.fontSizes = styledSystemTheme.typography;
-styledSystemTheme.fonts = styledSystemTheme.typography;
+const styledSystemTheme = {
+  breakpoints: ['40em', '52em', '64em'],
+  colors: {
+    primary: materialSystemTheme.palette.primary,
+    background: materialSystemTheme.palette.background,
+  },
+  fontSizes: materialSystemTheme.typography,
+  fonts: materialSystemTheme.typography,
+};
+
+const styledSystemCSSFunction = css({
```

Comment:
Should we move this part in the loop?

Reply:
Let’s try to see if there will be big difference

Reply:
I'm asking because it seems to be closer to how it then gets used in a Box. The props are provided at each render.

Reply:
Yeah, that's correct. I've moved, no big difference in perf.